build/linux/installer/conf/fluent-bit.conf

```diff
@@ -44,3 +44,16 @@
 #${MultilineEnabled}    Match oms.container.log.la.*
 #${MultilineEnabled}    multiline.key_content log
 #${MultilineEnabled}    multiline.parser #${MultilineLanguages}
+
+#${KubernetesFilterEnabled}[FILTER]
+#${KubernetesFilterEnabled}    Name                kubernetes
+#${KubernetesFilterEnabled}    Match               oms.container.log.la.*
+#${KubernetesFilterEnabled}    Merge_Log           Off
+#${KubernetesFilterEnabled}    Buffer_Size         0
+#${KubernetesFilterEnabled}    Use_Kubelet         true
```

Comment:
Lets validate the metadata and annotation based filtering with multi-line enabled.

Reply:
![image](https://github.com/microsoft/Docker-Provider/assets/25573844/87127ac3-cc3e-43bb-a153-db14eb063194)
verified